src/components/3_organisms/footer-deprecated.js

```diff
@@ -9,12 +9,12 @@ import Logo from "../../images/svgs/logo.svg";
 // import Github from "../../images/svgs/github.svg";
```

Comment:
Ignore the changes here, I was modifying this without realizing this is a deprecated component. Will likely delete before merge